bin/base_travis_integration_tests.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -e
+
+remove_lxd() {
+  sudo apt-get remove --yes --purge lxd lxd-client
+  sudo rm -Rf /var/lib/lxd
+}
+
+copy_deb_packages() {
+  mkdir deb-artifacts
+  cp *-debs/* deb-artifacts
+}
+
+create_pr_tar_file() {
+  cd $TRAVIS_BUILD_DIR/..
+  tar -zcf pr_source.tar.gz ubuntu-advantage-client
+  cp pr_source.tar.gz /tmp
+  ls -lh /tmp
+  cd $TRAVIS_BUILD_DIR
+}
+
+copy_deb_packages
```

Comment:
How we have side-effect functions being called just by sourcing this directory. I'd like to see all explicit call-sites in the scripts which source it. So we are not suprised by hidden behavior